guides/v2.3/release-notes/release-notes-2-3-3-open-source.md

```diff
@@ -75,7 +75,7 @@ The following upgrades to core platform components boost platform security and s
 *  Page load speeds have been improved by moving non-critical CSS elements to the bottom of the page. This enables the browser to render and display a storefront page more quickly. This setting is disabled by default, but you can enable it using **Stores** > **Configuration** > **Advanced** > **Developer** > **CSS Settings** > **Use CSS critical path**. For more information, see [CSS critical path documentation]({{ page.baseurl }}/frontend-dev-guide/css-topics/css-critical-path.html).
 
 <!--- MC-16887-->
-*  The `jQuery/ui` library has been refactored into separate widgets so that core modules load only the widgets they need. This update improves the performance of core storefront tasks including the loading of category, configurable product, home, and checkout pages.
+*  The `jQuery/ui` library has been refactored into separate widgets for frontend area so that core modules load only the widgets they need. This update improves the performance of core storefront tasks including the loading of category, configurable product, home, and checkout pages. As a module developer, you must update your code for storefront to remove `jquery/ui` dependency. Otherwise, your clients will see a performance degradation warning message in console.
```

Comment:
I'm not sure that the "for frontend area" assumption is 100% correct. Adminhtml area actively uses widgets such as modals and others that are based on `Magento_Ui -> jQuery/ui` widgets.